src/backend/marsha/core/models/video.py

```diff
@@ -21,6 +21,19 @@ class Video(BaseFile):
 
     RESOURCE_NAME = "videos"
 
+    active_shared_live_media = models.ForeignKey(
+        "SharedLiveMedia",
+        blank=True,
+        null=True,
+        on_delete=models.PROTECT,
+        related_name="active_video",
+        verbose_name=_("Video"),
+    )
+    active_shared_live_media_page = models.PositiveIntegerField(
```

Comment:
This property should also be set to null when the `shared_live_media` is deleted